src/js/load-package.ts

```diff
@@ -286,11 +299,16 @@ async function installPackage(
   let pkg: PackageData = API.lockfile_packages[name];
   if (!pkg) {
     pkg = {
+      name: "",
+      version: "",
       file_name: ".whl",
-      shared_library: false,
-      depends: [],
-      imports: [] as string[],
       install_dir: "site",
+      sha256: "",
+      package_type: "cpython_module",
```

Comment:
This condition is for installing a package from a URL, not installing a cpython_module. (Sorry, the existing code is a bit messy). So I think you need to set the `package_type` to "package".